static/js/copyPasteEvents.js

```diff
@@ -1,3 +1,5 @@
+var _ = require("underscore");
```

Comment:
Etherpad has a copy of underscore you can require: `var _ = require('ep_etherpad-lite/static/js/underscore');`